kernel-laws/src/main/scala/cats/kernel/laws/HashLaws.scala

```diff
@@ -0,0 +1,38 @@
+package cats.kernel
+package laws
+
+import org.typelevel.discipline.Laws
+import org.scalacheck._
+import org.scalacheck.Prop._
+
+object HashLaws {
+  def apply[A : Eq : Arbitrary : Cogen]: HashLaws[A] =
+    new HashLaws[A] {
+      def Equ = implicitly[Eq[A]]
+      def Arb = implicitly[Arbitrary[A]]
+      def Cog = implicitly[Cogen[A]]
+    }
+}
+
+/**
+ * @author Tongfei Chen
+ */
+trait HashLaws[A] extends Laws {
+
+  implicit def Equ: Eq[A]
+  implicit def Arb: Arbitrary[A]
+  implicit def Cog: Cogen[A]
+
+  def hash(implicit A: Hash[A]): HashProperties = new HashProperties(
+    name = "hash",
+    parent = None,
```

Comment:
Shouldn't this have `EqLaws` as parent?

Reply:
OrderLaws's parent is also None. But EqLaws are listed below in the `props`. I copied that behavior.

Reply:
seems like we should have `EqLaws` as a parent.

Reply:
I could find `EqLaws` in the `kernel-laws` package.